tests/Resources/Page/ODH/ODHDashboard/ODHDashboard.robot

```diff
@@ -593,3 +593,13 @@ RHODS Notification Drawer Should Not Contain
     Click Element    xpath=//*[contains(@class,'notification-badge')]
     Page Should Not Contain  text=${message}
     Click Element    xpath=//*[contains(@class,'notification-badge')]
+
+Check Doc And Quick Start Link In Enabled Page Tile
+    [Documentation]    Checks for the Doc and Quick Start Link in the tile of enabled page
+    ...                ${id} is the id of the tile
+    [Arguments]    ${id}
+    Click Button    //article[@id="${id}"]/div/div/div/button
+    ${href} =    Get Element Attribute    //a[contains(text(),"View documentation")]    href
+    Check HTTP Status Code    ${href}
+    Click Element    //a[contains(text(),"Open quick start")]
+    Wait Until Page Contains    Start
```

Comment:
could you create a kw in `ODHDashboardResources.resource` like `Page Should Contain QuickStart Sidebar` and use it here? 
You can find a similar kw in the same resource file, called `Page Should Not Contain QuickStart Sidebar`.

Besides, could you also close the quick start? You'll find keywords to handle it in above mentioned resource file